Rakefile

```diff
@@ -4,3 +4,15 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+
+
+# this task will get called before resque:pool:setup
+# and preload the rails environment in the pool manager
+task "resque:pool:setup" do
+  # close any sockets or files in pool manager
+  ActiveRecord::Base.connection.disconnect!
+  Resque::Pool.after_prefork do
+    ActiveRecord::Base.establish_connection
+  end
+end
```

Comment:
I don't totally understand this -- can you document where you got this from? Can we be confident this won't cause any problems on our existing infrastructure, where things seem to be working fine without it?  I wonder why we didn't need it before.

Reply:
The rationale given at https://devcenter.heroku.com/articles/ruby-resque-pool is:

"If you are using any connections in your application, such as connections to a database you need to be sure to disconnect and reconnect after the resque pool worker forks."

In other words, the rationale is to disconnect and reconnect to the database as part of the setup of the resque pool.

Just out of curiosity I'll try commenting it out and see if I get an error in Heroku, though.

Reply:
I think it's possible that is outdated info from heroku... I *think* it should have applied to our existing setup too the same, if it applied at all. But it is a bit confusing. resque is only barely supported software, which doesn't help. 

I'm not too worried about putting this in if we want to -- so long as we test it on staging first, since it will apply to our existing environment, AND leave a comment with the heroku docs URL that told us to do it, so someone can check later ot understand why, and see if heroku recs have changed at a later date.

Reply:
Looks like equivalent code is already present in `lib/tasks/resque.rake`; if I understand correctly, there's no need for this after all.

Reply:
Aha, good catch! Yep, we ahd already done what heroku suggested, just in a different file, ok!